pkg/webhook/cspc_test.go

```diff
@@ -740,10 +740,6 @@ func TestValidateCSPCUpdateRequest(t *testing.T) {
 
 				// Fetch blockdevice details and update
 				for _, bd := range getfakeBDs(test.updatedHostName, test.blockdevicePrefix, 7) {
-					_, err = f.wh.clientset.OpenebsV1alpha1().
-						BlockDevices(bd.Namespace).
-						Update(context.TODO(), bd, metav1.UpdateOptions{})
-				}
```

Comment:
any reason for removing this code snippet? It is required to test the following [case](https://github.com/openebs/cstor-operators/blob/b82dfa4ef36e6141c1d5dd00491d7344f054c521/pkg/webhook/cspc_test.go#L631).

Reply:
sorry removed unintentionally